src/patternfly/components/Page/docs/code.md

```diff
@@ -42,3 +45,4 @@ This component provides the basic chrome for a page, including sidebar, header,
 | `.pf-m-no-padding-mobile` | `.pf-c-page__main-section` | Modifies a main page section to have no padding on mobile. |
 | `.pf-m-fill` | `.pf-c-page__main-section` | Modifies a main page section to grow to fill the available vertical space. |
 | `.pf-m-no-fill` | `.pf-c-page__main-section` | Modifies a main page section to not grow to fill the available vertical space. |
+| `.pf-m-expanded` | `.pf-c-page__drawer` | Modifies the drawer for the expanded state. |
```

Comment:
Do you think its ok that we have an expanded modifier for the drawer but not the sidebar ?

Reply:
Hmm... we have `pf-m-expanded` for the sidebar, or are you referring to something else?

<img width="833" alt="Screen Shot 2019-07-08 at 4 28 14 PM" src="https://user-images.githubusercontent.com/35148959/60843881-79635080-a19d-11e9-96fd-f6eea5c30cf4.png">

Reply:
I missed that class, when there are two modifiers the same do we typically add them on one line in the docs like this:

`.pf-m-expanded` | `.pf-c-page__sidebar`, `.pf-c-page__drawer` | Modifies the sidebar and drawer for the expanded state.  |